nereid/nereid/tests/test_core/test_utils.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from nereid.core import utils
+
+
+@pytest.mark.parametrize(
+    "state, region, dirname, context, exp",
+    [
+        ("state", "region", None, None, {"test": True}),
+        ("state", "region", None, {"data_path": "string"}, {"data_path": "string"}),
+        (
+            "state",
+            "region",
+            "NotNone",
+            {"data_path": "string"},
+            {"data_path": "string"},
+        ),
+    ],
+)
+def test_get_request_context(state, region, dirname, context, exp):
```

Comment:
Having a hard time seeing what this is testing. Effectively, we're assertomg the response has the same key as the input?